libtextsecure/sendmessage.js

```diff
@@ -818,8 +669,15 @@ MessageSender.prototype = {
     const result = await sendPromise;
 
     // Sync the group message to our other devices
-    const encoded = textsecure.protobuf.DataMessage.encode(proto);
-    this.sendSyncMessage(encoded, timestamp, null, null, [], [], options);
+    const sentSyncMessageParams = {
+      timestamp,
+      dataMessage: proto,
+    };
+    const sentSyncMessage = new libsession.Messages.Outgoing.SentSyncMessage(
```

Comment:
Wouldn't it be easier if you just made the function take an object instead?
```ts
const sentSyncMessage = new libsession.Messages.Outgoing.SentSyncMessage({
    timestamp,
    dataMessage: proto
});
```

Reply:
I just separated the creation of the object from the function call itself, for improved readability